packages/mui-system/src/Grid/createGrid.tsx

```diff
@@ -122,6 +123,8 @@ export default function createGrid(
     const theme = useTheme();
     const themeProps = useThemeProps<typeof inProps & { component?: React.ElementType }>(inProps);
     const props = extendSxProp(themeProps) as Omit<typeof themeProps, 'color'> & GridOwnerState; // `color` type conflicts with html color attribute.
+    deleteLegacyGridProps(props, theme.breakpoints);
```

Comment:
please add a comment on what version to remove this line.

Reply:
Added